addons/storyshots/storyshots-puppeteer/src/index.js

```diff
@@ -65,6 +67,7 @@ export const imageSnapshot = (customConfig = {}) => {
       await page.goto(url, getGotoOptions({ context, url }));
       await beforeScreenshot(page, { context, url });
       image = await page.screenshot(getScreenshotOptions({ context, url }));
+      await beforeScreenshot({ image, context });
```

Comment:
shouldn't that be `afterScreenshot`? have you tested this?

Reply:
oh boy 🤦‍♂